src/VizzyUtilsContainer.tsx

```diff
@@ -19,11 +19,22 @@ import {
 declare var looker: Looker
 declare var LookerCharts: LookerChartUtils
 
-const vis: SteppedFunnelChart = {
+const vis: CustomViz = {
   // initial options applied to viz
   options: {
-    // AXES
-    label_left_axis: Vizzy.makeToggle("Axes", "Label Left Axis", false, 1),
+    view: Vizzy.makeList("Plot", "Json Inspector", "vizData", 
+    [
+      {"data": "data"},
+      {"element": "element"},
+      {"config": "config"},
+      {"queryResponse": "queryResponse"},
+      {"details": "details"},
+      {"done": "done"},
+      {"prepared data": "vizData"},
+    ],
+    0),
+    numDims: Vizzy.makeNumber("Plot", "Number of Dimensions", 1, 1),
```

Comment:
Vis config options should be snake_cased because these options are used in lookml definitions.